src/chats/components/infoPanel/conversationParticipantsAccordion/MemberComponentInfo.tsx

```diff
@@ -100,9 +100,11 @@ const MemberComponentInfo: FC<ParticipantsInfoProps> = ({ member, roomId }) => {
 					</Text>
 					<Padding top="extrasmall" />
 					{canSeeUsersPresence && (
-						<TextWithTooltip size="extrasmall" color="secondary">
-							{presenceLabel}
-						</TextWithTooltip>
+						<Tooltip label={presenceLabel} overflowTooltip>
+							<Text size="extrasmall" color="secondary">
```

Comment:
I notice that also the `memberName` could have the same issue but we forgot to add a tooltip for overflow 🙈, could you please add also in it in this PR? Thank you veeeery much